apis/projectcontour/v1/httpproxy.go

```diff
@@ -929,6 +929,12 @@ type DownstreamValidation struct {
 	// +kubebuilder:validation:MinLength=1
 	CACertificate string `json:"caSecret,omitempty"`
 
+	// Name of a Kubernetes secret that contains a pem encoded crl.
```

Comment:
I guess CRL file should allow [one or more](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto#envoy-v3-api-field-extensions-transport-sockets-tls-v3-certificatevalidationcontext-crl) PEM encoded CRLs concatenated together.